architecture description/dsl/plugins/org.polarsys.kitalpha.ad.viewpoint.dsl.as.diagram/model/vpdiagram.ecore

```diff
@@ -17,7 +17,7 @@
   </eClassifiers>
   <eClassifiers xsi:type="ecore:EClass" name="DiagramExtension" eSuperTypes="#//DiagramRepresentation">
     <eStructuralFeatures xsi:type="ecore:EReference" name="extented_diagram" lowerBound="1"
-        eType="ecore:EClass ../../org.eclipse.sirius.diagram/model/diagram.ecore#//description/DiagramDescription"/>
+        eType="ecore:EClass ../../org.eclipse.sirius.diagram.model//model/diagram.ecore#//description/DiagramDescription"/>
```

Comment:
Just coincidentally found it but is it normal to have double slashes here?

Reply:
It is not